test-network-function/common/env.go

```diff
@@ -86,6 +86,11 @@ func Intrusive() bool {
 	return !b
 }
 
+// GetOcDebugImageID is for running oc debug commands in a disconnected environment with a specific oc debug pod image mirrored
+func GetOcDebugImageID() string {
+	return os.Getenv("TNF_OC_DEBUG_IMAGE_ID")
```

Comment:
add code in the run-container.sh file to pass this new env variable to the container